.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Configure git
+      run: |
+        git config user.name "GitHub Actions Bot"
+        git config user.email "<>"
+
+    - name: Install bazelisk
+      run: |
+        curl -LO https://github.com/bazelbuild/bazelisk/releases/download/v1.7.4/bazelisk-linux-amd64
+        mkdir -p "${HOME}/bin"
+        mv bazelisk-linux-amd64 "${HOME}/bin/bazel"
+        chmod +x "${HOME}/bin/bazel"
+
+    - name: Check bindings generation
+      run: |
+        set -e
+        PATH=${HOME}/bin:$PATH ./hooks/pre-commit
+        # This might stage some files, if the bindings need to be regenerated.
```

Comment:
You could add a step that runs `git diff --exit-code HEAD --`.

Reply:
If we do this, then the check will fail on all PRs that haven't committed the updated go bindings- I'm not sure that we want to do that (in which case, maybe it would be better to `git commit --amend --no-edit`).